compose/production/nginx/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM nginx:1.19.0-alpine
+
+RUN rm /etc/nginx/conf.d/default.conf
+COPY nginx.conf /etc/nginx/conf.d
```

Comment:
probably should use Sebastian's approach of using the nginx image directly and setting the config via volumes

Reply:
I think this would be right if we were done customizing nginx, but I think there will be more configs we have to do to manage SSL and stuff, and so I think stubbing out a proper Dockerfile is actually a good idea.